tests/python/rest_api/test_tasks.py

```diff
@@ -553,8 +554,7 @@ def test_can_split_skeleton_tracks_on_jobs(self, jobs):
                             "frame": 0,
                             "shapes": [
                                 {"type": "points", "frame": 0, "points": [1.0, 2.0]},
-                                {"type": "points", "frame": 2, "points": [1.0, 2.0]},
-                                {"type": "points", "frame": 7, "points": [1.0, 2.0]},
+                                {"type": "points", "frame": 7, "points": [2.0, 4.0]},
```

Comment:
Please add a comment with the issue link in the test body, otherwise it's easy to forget about the problem during refactoring.